index.js

```diff
@@ -22,6 +22,29 @@ const removeMootSeparators = (string, separator) => {
 		.replace(new RegExp(`^${escapedSeparator}|${escapedSeparator}$`, 'g'), '');
 };
 
+const buildPatternSlug = options => {
+	// The allowSet contains some url-safe non-alphanumeric characters
+	const allowSet = new Set(['.', '#', '-', '_']);
```

Comment:
I'm confused why there is an allow list while the docs says you can use any character.

Reply:
I thought it would add a layer of safety, allowing maintainers to control what is preserved. I'll remove it, and it can be added in later, with docs too, if desired.